pysenteishon/static/js/pysenteishon.js

```diff
@@ -145,8 +148,25 @@ function NetworkInfo ($el) {
 }
 
 
+function Swipe ($el) {
+  this.$el = $el;
+  this.doDoing = false;
+
+  this.do = function(key) {press(key)};
+
+  this.$el.swipe({
+    swipe: function(event, direction, distance, duration, fingerCount, fingerData) {
+      // FIXME: "this.do" doesn't work here and it should :)
+      press(direction);
```

Comment:
Here, I tried to do

    this.do(direction);

but it says that `this.do` is not a function.

Sorry about this, I'm not good at js ;)